reference/ats/openapi.yaml

```diff
@@ -0,0 +1,671 @@
+          format: uuid
+          nullable: true
+        stage:
+          $ref: '#/components/schemas/Stage'
+          nullable: true
+        rejected_at:
+          type: string
+          format: date-time
+          nullable: true
+        rejected_reason:
+          type: object
+          nullable: true
+          properties:
+            text:
+              type: string
+              nullable: true
+      x-examples:
+        success:
+          id: 497f6eca-6276-4993-bfeb-53cbbbba6f08
+          candidate_id: 2f28f791-f0fd-4155-a356-c24e996beeda
+          job_id: 453bd7d7-5355-4d6d-a38e-d9e7eb218c3f
+          offer_id: d5a7a5b7-a4a3-49e7-9c69-b44d2cbe15cf
+          stage:
+            id: bf25a12f-4544-4b77-a0c9-5726e16b7505
+            job_id: null
+            name: Phone Screen
+          rejected_at: '2019-08-24T14:15:22Z'
+          rejected_reason:
+            text: Underqualified
+    Job:
+      title: Job
+      x-stoplight:
+        id: bfuo8aqn6gz2v
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        name:
+          type: string
+          nullable: true
+        status:
+          type: string
+          enum:
+            - open
+            - closed
+            - on_hold
+            - draft
+            - archived
+          nullable: true
+        department:
+          type: object
+          properties:
+            name:
+              type: string
+              nullable: true
+        created_at:
+          type: string
+          format: date-time
+          nullable: true
+        closed_at:
+          type: string
+          format: date-time
+          nullable: true
+        hiring_team:
+          type: object
+          properties:
+            hiring_managers:
+              type: array
+              nullable: true
+              items:
+                type: object
+                properties:
+                  name:
+                    type: string
+            recruiters:
+              type: array
+              nullable: true
+              items:
+                type: object
+                properties:
+                  name:
+                    type: string
+      x-examples:
+        success:
+          id: 497f6eca-6276-4993-bfeb-53cbbbba6f08
+          name: Office Manager
+          status: open
+          department:
+            name: People
+          created_at: '2019-08-24T14:15:22Z'
+          closed_at: '2019-08-24T14:15:22Z'
+          hiring_team:
+            hiring_managers:
+              - name: Michael Glen
+            recruiters:
+              - name: Chris Pine
+    Stage:
+      title: Stage
+      x-stoplight:
+        id: gul2e0glhwi6o
+      type: object
+      x-examples:
+        example-1:
+          id: 497f6eca-6276-4993-bfeb-53cbbbba6f08
+          job_id: null
+          name: Phone Screen
+      properties:
+        id:
+          type: string
+          format: uuid
+        job_id:
```

Comment:
Don't think you need this cause the application responses include `job_id`

Reply:
+1

Reply:
it is needed. read the description:
"The job id that this stage applies to, if applicable. Not all systems enumerate stages specific to jobs."
A stage can be specific to a job